tests/basic_relay.go

```diff
@@ -85,6 +86,8 @@ func BasicRelay(network *network.LocalNetwork, teleporter utils.TeleporterTestIn
 		fundedAddress,
 	)
 
+	time.Sleep(10 * time.Second)
```

Comment:
why this custom sleep? `RelayBasicMessage` should be blocking right?

Reply:
This was a test artifact. Removed now.